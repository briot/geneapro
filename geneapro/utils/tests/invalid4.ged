0 HEAD
1 SOUR GRAMPS
1 DATE 10 NOV 2008
1 SUBM @SUBM@
1 GEDC
2 VERS 5.5
2 FORM LINEAGE-LINKED
1 CHAR UTF-8
