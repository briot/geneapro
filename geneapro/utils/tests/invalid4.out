<dir>/invalid4.ged:8 Missing 1 occurrences of TRLR in file
